pkg/sql/catalog/systemschema/system.go

```diff
@@ -1215,29 +1225,65 @@ var (
 // suggestions on writing and maintaining them.
 var (
 	// LeaseTable is the descriptor for the leases table.
-	LeaseTable = makeSystemTable(
-		LeaseTableSchema,
-		systemTable(
-			catconstants.LeaseTableName,
-			keys.LeaseTableID,
-			[]descpb.ColumnDescriptor{
-				{Name: "descID", ID: 1, Type: types.Int},
-				{Name: "version", ID: 2, Type: types.Int},
-				{Name: "nodeID", ID: 3, Type: types.Int},
-				{Name: "expiration", ID: 4, Type: types.Timestamp},
-			},
-			[]descpb.ColumnFamilyDescriptor{
-				{Name: "primary", ID: 0, ColumnNames: []string{"descID", "version", "nodeID", "expiration"}, ColumnIDs: []descpb.ColumnID{1, 2, 3, 4}},
-			},
-			descpb.IndexDescriptor{
-				Name:                "primary",
-				ID:                  1,
-				Unique:              true,
-				KeyColumnNames:      []string{"descID", "version", "expiration", "nodeID"},
-				KeyColumnDirections: []catpb.IndexColumn_Direction{catpb.IndexColumn_ASC, catpb.IndexColumn_ASC, catpb.IndexColumn_ASC, catpb.IndexColumn_ASC},
-				KeyColumnIDs:        []descpb.ColumnID{1, 2, 4, 3},
-			},
-		))
+	LeaseTable = func() SystemTable {
+		if TestSupportMultiRegion() {
+			return makeSystemTable(
+				MRLeaseTableSchema,
+				systemTable(
+					catconstants.LeaseTableName,
+					keys.LeaseTableID,
+					[]descpb.ColumnDescriptor{
+						{Name: "descID", ID: 1, Type: types.Int},
+						{Name: "version", ID: 2, Type: types.Int},
+						{Name: "nodeID", ID: 3, Type: types.Int},
+						{Name: "expiration", ID: 4, Type: types.Timestamp},
+						{Name: "crdb_region", ID: 5, Type: types.Bytes},
+					},
+					[]descpb.ColumnFamilyDescriptor{
+						{
+							Name:        "primary",
+							ID:          0,
+							ColumnNames: []string{"descID", "version", "nodeID", "expiration", "crdb_region"},
+							ColumnIDs:   []descpb.ColumnID{1, 2, 3, 4, 5},
+						},
+					},
+					descpb.IndexDescriptor{
+						Name:           "primary",
+						ID:             1,
```

Comment:
The multi-region index should use ID 2 if we want to support migrating from the old schema to the new schema using a protocol resembling a standard schema change.

Reply:
Done.